_data/contributors.yml

```diff
@@ -240,6 +240,34 @@
 #                                     2024                                     #
 ################################################################################
 
+- name: Khushiyant
```

Comment:
This needs to move before `- name: "This could be you!"`.

Reply:
Sure